en/pause-sync-of-tidb-cluster.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Pause Sync of a TiDB Cluster in Kubernetes
+summary: Introduce how to pause sync of a TiDB cluster in Kubernetes
+---
+
+# Pause Sync of a TiDB Cluster in Kubernetes
+
+This document introduce how to pause sync of a TiDB cluster in Kubernetes with configuration.
+
+## What is sync in TiDB Operator
+
+In TiDB Operator, controller regulates the state of the TiDB cluster in Kubernetes. The controller constantly compares the desired state recorded in the `TidbCluster` object with the actual state of the TiDB cluster. This process is referred to as **sync** generally. For more details, refer to [TiDB Operator Architecture](architecture.md).
+
+## Use scenarios
+
+Here are some cases where you might need to pause sync of a TiDB cluster in Kubernetes.
+
+- Avoiding unexpected rolling update
+
+    In order to prevent the compatibility of the new version TiDB Operator from affecting the TiDB clusters, you can pause sync of TiDB clusters before updating the TiDB Operator. After updating TiDB Operator, resume sync of TiDB clusters one by one or resume sync of TiDB clusters at a specified time, so as to observe the impact of TiDB Operator rolling update on the cluster.
+
+- Avoid multiple rolling restarts
+
+    In some cases, the configuration of a TiDB cluster may be modified several times over a period of time, but you do not want to restart the TiDB cluster many times. In order to avoid multiple rolling restarts, you can pause sync of a TiDB cluster first. During this period, any modification to configuration of the TiDB cluster will not take effect. After the cluster configuration modification is completed, resuming sync of TiDB Cluster. At this time, multiple configuration changes during the sync pausing can be applied in one time rolling restart.
+
+- Maintenance window
+
+    In some situations, you can update or restart TiDB cluster only during a maintenance window. When outside the maintenance window, you can pause sync of the TiDB cluster, so that any modification to spec does not take effect. When inside the maintenance window, you can resume sync of the TiDB cluster to allow TiDB cluster rolling update or restart.
```

Comment:
```suggestion
    In some situations, you can update or restart the TiDB cluster only during a maintenance window. When outside the maintenance window, you can pause sync of the TiDB cluster, so that any modification to the specs does not take effect. When inside the maintenance window, you can resume sync of the TiDB cluster to allow TiDB cluster to rolling update or restart.
```

Reply:
fixed